src/utils/dom.js

```diff
@@ -3,3 +3,17 @@ export const htmlToElement = html => {
   template.innerHTML = html
   return template.content.firstChild
 }
+
+export const bindEvents = (root, context, ...eventNames) => {
+  eventNames.forEach(eventName => {
+    const attributeName = `data-bind-${eventName}`
+    Array.from(root.querySelectorAll(`[${attributeName}]`)).forEach(
+      clickable => {
```

Comment:
@francesco-strazzullo using the name `clickable` is a little bit confused for me, what do you think to use something like `bindable` instead?

Reply:
Because, if I understand, this code works for all events non only click event. Or not?

Reply:
Yeah it's a typo

Reply:
I will change it with element

Reply:
I did it here: https://github.com/e-xtrategy/framework-compass-chart-editor/pull/6/commits/924bf7a4df0c1fcade6786e8bcfe35e387575bbe